lib/agent0/agent0/hyperdrive/interactive/interactive_hyperdrive_test.py

```diff
@@ -590,3 +619,55 @@ def test_policy_config_forgotten(chain: LocalChain):
         policy=Zoo.random,
     )
     assert alice.agent.policy is not None
+
+
+@pytest.mark.anvil
+def test_share_price_quincunx(chain: Chain):
+    """Share price when compounding by quincunx."""
```

Comment:
can you add descriptions explaining "compounding by quincunx" and "compounding by annus"?

Reply:
maybe also a quick note on _why_ you're writing these two tests